tests/WebexMeeting.e2e.js

```diff
@@ -75,4 +75,13 @@ describe('Meeting Widget', () => {
     expect(MeetingPage.unmuteAudioBtn).toBeVisible();
     MeetingPage.unloadWidget();
   });
+
+  it('leave the meeting', () => {
+    MeetingPage.loadWidget(meetingDestination);
+    MeetingPage.joinMeetingBtn.click();
+    MeetingPage.waitingForOthers.waitForDisplayed({timeout: 10000});
+    MeetingPage.leaveMeetingBtn.click();
+    expect($('.wxc-meeting')).toHaveTextContaining("You've successfully left the meeting");
```

Comment:
This should be part of the `MeetingPage`, similar to how it is [done for `waiting for others`](https://github.com/webex/widgets/pull/58/files#diff-f5d4a1540edc79b8cc150656aaa0a676d13340692408b2aa279a92c96ec973e9L65)